lib/android-helpers.js

```diff
@@ -890,5 +891,46 @@ helpers.isEmulator = function isEmulator (adb, opts) {
 helpers.bootstrap = Bootstrap;
 helpers.unlocker = unlocker;
 
+const ARGS_CONSTRAINTS = {
+  reboot: {
+    isBoolean: true,
+  },
+  chromeDriverPort: {
+    isNumber: true,
+  },
+  chromedriverExecutable: {
+    isString: true,
+  },
+  suppressKillServer: {
+    isBoolean: true,
+  },
+};
+
+helpers.parseKnownArgs = function (allArgs, knownArgNames) {
```

Comment:
are we going to reimplement these functions in every driver? if so, we'd be better off putting this in appium-support or something like that (with a TODO that it should really be in basedriver once appium 2 is out). also, i don't see where these functions are used?

Reply:
Would be better to put them in appium-support I agree, and they are used by the android drivers: uiautomator2, espresso.